### DISCUSSION TITLE
How can I use the API to obtain what is shown here: https://marine.weather.gov/MapClick.php?zoneid=PZZ545&TextType=1

### DISCUSSION THREAD

**Discussion by rtlemos** [weather-gov/api#304] [Q&A]
**NOTE**
Please do not report operational issues (such as outages) here. We can not support these issues via this forum. Please report these as described here: https://weather-gov.github.io/api/reporting-issues

**Please describe what you're trying to do.**
A clear and concise description of what your issue is. Ex. I'm looking for forecast data...

I'm trying to fetch the forecast shown in [marine.weather.gov ](https://marine.weather.gov/MapClick.php?zoneid=PZZ545&TextType=1)using the API, but I see nothing when I do
https://api.weather.gov/zones/forecast/PZZ545
Is there another access point?

**Comment by KTibow** [comment]
Some stuff isn't available through the API...

**Comment by scadergit** [comment]
Correct, marine is not implemented in the API yet, as they don't send us all the grids yet.

**Comment by rtlemos** [comment]
Thank you both! Is there a way to register for updates? (other than "Watch releases")

**Comment by scadergit** [comment]
Sorry, the current way is monitoring service change noticies (SCN) at https://www.weather.gov/notification. Lots of non-API notices in there...we're working on a better channel for API-related, but nothing official yet.
